src/main/java/io/javalin/core/util/RateLimitUtil.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Javalin - https://javalin.io
+ * Copyright 2017 David Åse
+ * Licensed under Apache 2.0: https://github.com/tipsy/javalin/blob/master/LICENSE
+ */
+
+package io.javalin.core.util
+
+import io.javalin.http.Context
+import io.javalin.http.HttpResponseException
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.Executors
+import java.util.concurrent.TimeUnit
+
+object RateLimitUtil {
+    val handlerToIpToRequestCount = HashMap<String, ConcurrentHashMap<String, Int>>()
+
+    init {
+        Executors.newSingleThreadScheduledExecutor().scheduleAtFixedRate({
+            handlerToIpToRequestCount.forEach { (_, ipRequestCount) -> ipRequestCount.clear() }
+        }, 5, 1, TimeUnit.SECONDS) // the initial delay is there to ensure deterministic tests
```

Comment:
Reading this code without an editor it's not clear if it's every 1 second with 5 second delay or vica versa. I guess you can't use named variables with the underlying Java code?

Reply:
I see the comment below answers my question.